ui2/src/pages/clearing/Members.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { withRouter, RouteComponentProps, NavLink } from 'react-router-dom';
+import { useParty, useStreamQueries } from '@daml/react';
+import { AssetDeposit } from '@daml.js/da-marketplace/lib/DA/Finance/Asset';
+import { AssetSettlementRule } from '@daml.js/da-marketplace/lib/DA/Finance/Asset/Settlement';
+import { getName } from '../../config';
+import { Service } from '@daml.js/da-marketplace/lib/Marketplace/Clearing/Service';
+import { MemberStanding } from '@daml.js/da-marketplace/lib/Marketplace/Clearing/Model';
+import { ServicePageProps } from '../common';
+import { Header } from 'semantic-ui-react';
+import Tile from '../../components/Tile/Tile';
+import StripedTable from '../../components/Table/StripedTable';
+import MarginCallModal from './MarginCallModal';
+import MTMCalculationModal from './MTMCalculationModal';
+import { ArrowRightIcon } from '../../icons/icons';
+
+const ClearingMembersComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> = ({
+  history,
+  services,
+}) => {
+  const party = useParty();
+
+  const accounts = useStreamQueries(AssetSettlementRule).contracts;
+  const deposits = useStreamQueries(AssetDeposit).contracts;
+  const standings = useStreamQueries(MemberStanding).contracts;
+  const ccpDeposits = deposits.filter(
+    d => d.payload.account.id.label === services[0].payload.ccpAccount.id.label
+  );
+
+  const formatter = new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+  });
```

Comment:
This is currently a placeholder since we're only supporting USD with baymarkets at the moment, but I'll clean it up in the next go to do it dynamically anyway.

Reply:
Makes sense, thanks @alexgraham-da !